### PR TITLE
Add Microsoft.Scheduler schema

### PR DESCRIPTION
**Pull request by daschult**

### CODE REVIEW

File: schemas/2016-03-01/Microsoft.Scheduler.json [hunk 1]
```diff
@@ -0,0 +1,999 @@
...
+            "2016-03-01"
+          ]
+        },
+        "name": {
+          "type": "string",
+          "description": "Gets or sets the job collection resource name."
+        },
+        "location": {
+          "type": "string",
+          "description": "Gets or sets the storage account location."
```
Comment (kevinlam1) [inline schemas/2016-03-01/Microsoft.Scheduler.json]: I think this is a cut and paste error and should say job collection.

File: schemas/2014-04-01-preview/deploymentTemplate.json [hunk 2]
```diff
@@ -120,6 +120,8 @@
                   { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_SOA"},
                   { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_SRV"},
                   { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_TXT"},
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-01/Microsoft.Scheduler.json#/resourceDefinitions/jobCollections"},
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-01/Microsoft.Scheduler.json#/resourceDefinitions/jobCollections_jobs"},
```
Comment (kevinlam1) [inline schemas/2014-04-01-preview/deploymentTemplate.json]: Doesn't only the top level resources get exposed here?  A job is not a registered resource.

File: schemas/2015-01-01/deploymentTemplate.json [hunk 3]
```diff
@@ -118,7 +118,9 @@
                   { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_PTR"},
                   { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_SOA"},
                   { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_SRV"},
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_TXT"}
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-04-01/Microsoft.Network.json#/resourceDefinitions/dnszones_TXT"},
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-01/Microsoft.Scheduler.json#/resourceDefinitions/jobCollections"},
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-03-01/Microsoft.Scheduler.json#/resourceDefinitions/jobCollections_jobs"}
```
Comment (kevinlam1) [inline schemas/2015-01-01/deploymentTemplate.json]: Don't only top level resources get exposed here?  Job is not an ARM registered resource (only job collection).

File: schemas/2016-03-01/Microsoft.Scheduler.json (hunk 1 above)
Comment (daschult) [inline schemas/2016-03-01/Microsoft.Scheduler.json]: Fixed.

**Comment by daschult** [comment]
Can we merge this pull request?

**Comment by kevinlam1** [comment]
+Ping
Looks good to me.

-Kevin

File: schemas/2016-03-01/Microsoft.Scheduler.json [hunk 4]
```diff
@@ -0,0 +1,999 @@
...
+              "$ref": "#/definitions/JobCollectionProperties"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Gets or sets the job collection properties."
+        }
+      },
+      "resources": {
```
Comment (TianoMS) [inline schemas/2016-03-01/Microsoft.Scheduler.json]: "resources" should not be here.

**Pull request merged**